pip/pip-375.md

```diff
@@ -0,0 +1,32 @@
+# PIP-375: Expose the Admin client configs: readTimeout, requestTimeout, connectionTimeout
+
+# Background knowledge
+
+- `pulsar-admin sinks create` will upload files to brokers, but if the file is large and the network is slow, it may cost a long time.
+- The default value of `requestTimeout` of Admin API is `60s`, after the fix https://github.com/apache/pulsar/pull/23128, it was changed to `300s`.
```

Comment:
regarding "it was changed to `300s`", where did that exactly happen?

Reply:
The default value of `requestTimeout` is `300s` and the default value of `readTimeout` is `60s`, before #23128 , the admin client used an in-correct variable (`readTimeout = 60s`), after that it used a correct variable (`requestTimeout=300s`)

Reply:
Here it is: https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminImpl.java#L76

Reply:
Do you mean to say that the default *was* 60s before the change in PR 23128, which made the default value 300s?
Why was this increase in the value a problem? Or was it a problem?